app/views/rails_admin/main/import_failure.js.erb

```diff
@@ -0,0 +1,5 @@
+setTimeout(function() {
+  $('.govuk-form-group').addClass('govuk-form-group--error')
+  $('#upload-file-error span.message').html('An error occourred during the file upload. Please check if your file is an Excel and correctly formatted.')
+  $('#upload-file-success').addClass('govuk-visually-hidden')
+}, 500)
```

Comment:
Hi @cacquani, I'm confused about this 0.5s timeout. Is there a reason not to just display this message immediately on response?

Reply:
The reason is that in case of error (which is immediate) it's just really annoying to see this thing flicker for a millisecond and disappear before you can make out what it is.
It's also on success to match the `ajaxStop` delay in `app/views/rails_admin/main/import.html.erb:15-20`